packages/firestore/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "outDir": "dist"
   },
   "exclude": [
+    "scripts/**/*",
```

Comment:
Is it for performance or something else?

Reply:
Yes, otherwise it parses all the TS files in this tree.